rpm/podman.spec

```diff
@@ -187,7 +191,9 @@ when `%{_bindir}/%{name}sh` is set as a login shell or set as os.Args[0].
 Summary: Metapackage for setting up %{name} machine
 Requires: %{name} = %{epoch}:%{version}-%{release}
 Requires: gvisor-tap-vsock
+%if %{defined qemu}
```

Comment:
Perhaps you want `qemu-kvm`? Publishing podman-machine in RHEL without QEMU otherwise feels rather moot?